webapp/components/hooks/usePost.js

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from 'react'
+import { useDispatch } from 'react-redux'
+
+import { asyncScheduler, Subject } from 'rxjs'
+import { switchMap, throttleTime } from 'rxjs/operators'
+import { ajax } from 'rxjs/ajax'
+import { fromAjax } from '@webapp/utils/rxjs'
+
+const createAjaxSubject = ({ dispatch, throttle }) => {
+  const fetchSubject = new Subject()
+  return fetchSubject.pipe(
+    throttleTime(throttle, asyncScheduler, { leading: false, trailing: true }),
```

Comment:
what is exactly the meaning of leading and trailing in this context.

I read [that](https://rxjs-dev.firebaseapp.com/api/operators/throttleTime) and the configuration looks like the double click example which listens to the second click of every tuple of click which happens between 400ms but I think that I am not able to understand it yet

Reply:
it takes the trailing event, not the leading one.
e.g. [e1,e2,e3] with leading true, e1 goes through, with trailing true, e3 goes through